entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+python manage.py makemigrations --no-input
+
+python manage.py migrate --no-input
```

Comment:
You don't need these commands, `makemigrations` should be run manually and created migrations should be added to git and migrate should be executed manually when you actually need it by running `docker-compose run web migrate`. `makemigrations` can be run the same way.

Reply:
I guess this is not fixed.